ext/repo_comps.h

```diff
@@ -5,4 +5,9 @@
  * for further information
  */
 
+#include <stdbool.h>
+
 extern int repo_add_comps(Repo *repo, FILE *fp, int flags);
+
+const bool COMPS_DEFAULT_USERVISIBLE = true;
+const bool COMPS_DEFAULT_DEFAULT = false;
```

Comment:
Is this something that should be exposed in the API or should be internal to the code? If the latter, shouldn't this be in the `repo_comps.c` file?

Reply:
It doesn't need to be exposed, so I moved it to ``repo_comps.c``.